Mapping/ClassMetadata.php

```diff
@@ -106,6 +115,22 @@ public function getIri()
     }
 
     /**
+     * @return string
+     */
+    public function getIdentifier()
+    {
+        return $this->identifier;
```

Comment:
The order is required because of the priority of identifying the identfier (doctrine, and after a custom identifier; if we introduce it we need to be sure that the doctrineLoader doesn't erase it ... )

Ok I understand why you don't want our own Identifier annotation, yes I we need it we can implement it in our project, but provide a default implementation can be cool, no ?